test/helpers.py

```diff
@@ -176,7 +181,10 @@ def _batch_sde_solve(
         stepsize_controller=controller,
         saveat=saveat,
     )
-    return sol.ys, sol.stats["num_accepted_steps"]
+    steps = sol.stats["num_accepted_steps"]
+    if isinstance(solver, diffrax.HalfSolver):
+        steps *= 3
```

Comment:
I don't think this is fair! Different solvers anyway have different per-step costs; if the desire is to normalize over that then that should be done elsewhere across all kinds of solvers.

Reply:
You're right this doesn't have to be present in this branch.